docs/react/getting-started.en-US.md

```diff
@@ -0,0 +1,174 @@
+
+```html
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <!-- import stylesheet -->
+    <link rel="stylesheet" href="/index.css">
+    <!-- Polyfills -->
+    <!--[if lt IE 10]>
+    <script src="https://as.alipayobjects.com/g/component/??console-polyfill/0.2.2/index.js,es5-shim/4.5.7/es5-shim.min.js,es5-shim/4.5.7/es5-sham.min.js,html5shiv/3.7.2/html5shiv.min.js,media-match/2.0.2/media.match.min.js"></script>
+    <![endif]-->
+  </head>
+  <body>
+  </body>
+  <!-- import common dependencies -->
+  <script src="/common.js"></script>
+  <!-- import entry file -->
+  <script src="/index.js"></script>
+</html>
+```
+
+You may meet problems like [#28](https://github.com/ant-tool/atool-build/issues/28) and [#858](https://github.com/ant-design/ant-design/issues/858) for `babel@6.x` doesn't support IE8.
+
+[antd-init](http://github.com/ant-design/antd-init) had solved those problems, and you can refer this [webpack config](https://github.com/ant-design/antd-init/blob/f5fb9479ca973fade51fd6754e50f8b3fafbb1df/boilerplate/webpack.config.js#L4-L8)。
+
+> More about how to use React in IE8: https://github.com/xcatliu/react-ie8
+
+## Customized Work Flow
+
+If you want to customized you own work flow, we recommend to use [webpack](http://webpack.github.io/) to build and debug code.
+
+Also, you can use any [scaffold](https://github.com/enaqx/awesome-react#boilerplates) in React ecosystem in your work flow. If you meet problems, you can use our [webpack config](https://github.com/ant-tool/atool-build/blob/master/src/getWebpackCommonConfig.js) and [modify it](http://ant-tool.github.io/webpack-config.html).
+
+There are some [scaffold](https://github.com/ant-design/ant-design/issues/129) which had integrated antd, you can try and contribute.
+
+## Import on Demand
+
+If we import a component like this `import { Button } from 'antd';`, then all the components of antd will be imported. But, we can import component on demand:
+
+```jsx
+import Button from 'antd/lib/button';
+```
+
+If you use `babel`, we recommend to use [babel-plugin-antd](https://github.com/ant-design/babel-plugin-antd). This plugin will convert the following code to the above form:
+
+```jsx
+import { Button } from 'antd';
+```
+
+And, this plugin can load style on demand, too. See: [style](https://github.com/ant-design/babel-plugin-antd#usage).
+
+## Customization
+
+- [Customize Theme](https://github.com/ant-design/antd-init/tree/master/examples/customize-antd-theme)
+- [Local Iconfont](https://github.com/ant-design/antd-init/tree/master/examples/local-iconfont)
+
+## Tips
+
+- You can use any `npm` modules.
+- We recommend to write code in [ES2015](http://babeljs.io/blog/2015/06/07/react-on-es6-plus), for `babel` had be integrated in our work flow.
```

Comment:
- We recommend to write code in [ES2015](http://babeljs.io/blog/2015/06/07/react-on-es6-plus) as `babel` has been integrated in our work flow.